### PR TITLE
Issue 506 Added Example using FedProx

### PR DESCRIPTION
**Pull request by bjklemme-intel**

### CODE REVIEW

File: openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# MedMNIST 2D Classification Using FedProx Optimizer Tutorial
+
+![MedMNISTv2_overview](https://raw.githubusercontent.com/MedMNIST/MedMNIST/main/assets/medmnistv2.jpg)
+
```
Comment (psfoley) [inline openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/README.md:4]: More information about FedProx / a link to the paper should also be included here

File: openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/workspace/Pytorch_FedProx_MedMNIST_2D.ipynb [hunk 2]
```diff
@@ -0,0 +1,628 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "26fdd9ed",
+   "metadata": {},
+   "source": [
+    "# Federated MedMNIST2D "
```
Comment (psfoley) [inline openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/workspace/Pytorch_FedProx_MedMNIST_2D.ipynb]: The title should be changed to `Federated MedMNIST2D Using FedProx Aggregation Algorithm

File: openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/workspace/Pytorch_FedProx_MedMNIST_2D.ipynb [hunk 3]
```diff
@@ -0,0 +1,628 @@
...
+    "sample, target = dummy_shard_dataset[0]\n",
+    "f\"Sample shape: {sample.shape}, target shape: {target.shape}\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cc0dbdbd",
+   "metadata": {},
+   "source": [
+    "## Describing FL experimen"
```
Comment (psfoley) [inline openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/workspace/Pytorch_FedProx_MedMNIST_2D.ipynb]: ## Describing FL experiment

File: openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/README.md (hunk 1 above)
Comment (bjklemme-intel) [reply openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/README.md:4]: I made this change.

File: openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/workspace/Pytorch_FedProx_MedMNIST_2D.ipynb (hunk 3 above)
Comment (bjklemme-intel) [reply openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/workspace/Pytorch_FedProx_MedMNIST_2D.ipynb]: I made this change.

File: openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/workspace/Pytorch_FedProx_MedMNIST_2D.ipynb (hunk 2 above)
Comment (bjklemme-intel) [reply openfl-tutorials/interactive_api/PyTorch_FedProx_MNIST/workspace/Pytorch_FedProx_MedMNIST_2D.ipynb]: I made this change. Thank You!

**Comment by bjklemme-intel** [comment]
can we go ahead and merge this?

**Review by psfoley — APPROVED** [review]

**Pull request merged**